### PR TITLE
Code changes to make Windows build work..

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary

- These changes to header includes, types, and system calls are needed to make RAJA Perf compile with MSVC
- While working on a GitHub Actions Windows build and test pipeline, I hit compilation issues.
- These changes are necessary to get that CI job, which will be included in a subsequent PR, to work. These changes will not be tested in the current CI configuration, but do not affect any non-Windows builds.

### CODE REVIEW

File: src/common/OutputUtils.cpp [hunk 1]
```diff
@@ -19,8 +19,12 @@
 #include<fstream>
 #include<sstream>
 
+#if defined(_WIN32)
+#nclude "WindowsTypes.hpp"
```
Comment (MrBurmark) [inline src/common/OutputUtils.cpp:23]:
```suggestion
#include "WindowsTypes.hpp"
```

**Review by MrBurmark — APPROVED** [review]

File: src/common/WindowsTypes.hpp [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+///
+/// Type and value definitions to support Windows builds.
+/// Windows has different syntax and does not support all POSIX modes
+/// that Linux systems do.
+///
+
+#ifndef RAJAPerf_WindowsTypes_HPP
+#define RAJAPerf_WindowsTypes_HPP
+
+#include<sys/types.h>
```
Comment (adayton1) [inline src/common/WindowsTypes.hpp:18]: Nitpick, but it would be nice to have a space between `#include` and the file name.

**Review by adayton1 — APPROVED** [review]
LGTM

File: src/common/WindowsTypes.hpp (hunk 2 above)
Comment (rhornung67) [reply src/common/WindowsTypes.hpp:18]: We currently do both (space and no space) throughout the code. After I get the azure and GH Actions CI PRs in, I will attempt to set up a style check action. That should be a good way to make things like this consistent.

**Pull request merged**